Makefile

```diff
@@ -40,9 +40,6 @@ swig: sdk
 build-module: swig
 	mkdir -p bin && CGO_LDFLAGS=${CGO_LDFLAGS} go build -o bin/rplidar-module module/main.go
 
-build-server: swig
```

Comment:
[note to self] removing unnecessary code: https://viaminc.slack.com/archives/C036N5GJFLM/p1681851423418369